src/libs/antares/study/area/list.cpp

```diff
@@ -305,6 +305,21 @@ static bool AreaListSaveToFolderSingleArea(const Area& area, Clob& buffer, const
         buffer.clear() << folder << SEP << "input" << SEP << "renewables" << SEP << "series";
         ret = area.renewable.list.saveDataSeriesToFolder(buffer) && ret;
     }
+
+    // Short term storage
+
+    // save sts in list.ini for this area
+    buffer.clear() << folder << SEP << "input" << SEP << "st-storage" << SEP << "clusters"
+        << SEP << area.id;
+    IO::Directory::Create(buffer);
```

Comment:
Please place this directory creation in `shortTermStorage.saveToFolder`, line below.